src/plugins/ui_feature/autofill.ts

```diff
@@ -407,13 +522,8 @@ export class AutofillPlugin extends UIPlugin {
       : position[second].value;
   }
 
-  private autoFillMerge(
-    originCol: HeaderIndex,
-    originRow: HeaderIndex,
-    col: HeaderIndex,
-    row: HeaderIndex
-  ) {
-    const sheetId = this.getters.getActiveSheetId();
+  private autofillMerge(sheetId: UID, data: Omit<AutoFillCellCommand, "type">) {
```

Comment:
nitpick: I'd create a type `AutofillCellData` at this point, `Omit<AutoFillCellCommand, "type">` is used like 7 times